python/cugraph/tests/utils.py

```diff
@@ -40,46 +40,61 @@
 #
 # Datasets
 #
-DATASETS_UNDIRECTED = ["../datasets/karate.csv", "../datasets/dolphins.csv"]
+
+
+RAPIDS_DATASET_ROOT_DIR = os.getenv("RAPIDS_DATASET_ROOT_DIR", "../datasets")
+
+DATASETS_UNDIRECTED = [PurePath(RAPIDS_DATASET_ROOT_DIR)/f for
+                       f in ["karate.csv", "dolphins.csv"]]
 
 DATASETS_UNDIRECTED_WEIGHTS = [
-    "../datasets/netscience.csv",
+    PurePath(RAPIDS_DATASET_ROOT_DIR)/"netscience.csv"
 ]
 
-DATASETS_UNRENUMBERED = ["../datasets/karate-disjoint.csv"]
+DATASETS_UNRENUMBERED = [PurePath(
+    RAPIDS_DATASET_ROOT_DIR)/"karate-disjoint.csv"
+]
 
-DATASETS = [
-    "../datasets/karate-disjoint.csv",
-    "../datasets/dolphins.csv",
-    "../datasets/netscience.csv",
+DATASETS = [PurePath(RAPIDS_DATASET_ROOT_DIR)/f for f in [
+    "karate-disjoint.csv",
+    "dolphins.csv",
+    "netscience.csv"]
 ]
+
+
 #            '../datasets/email-Eu-core.csv']
 
 STRONGDATASETS = [
-    "../datasets/dolphins.csv",
-    "../datasets/netscience.csv",
-    "../datasets/email-Eu-core.csv",
+    PurePath(RAPIDS_DATASET_ROOT_DIR)/f for f in [
+        "dolphins.csv",
+        "netscience.csv",
+        "email-Eu-core.csv"]
 ]
 
+
 DATASETS_KTRUSS = [
-    ("../datasets/polbooks.csv", "../datasets/ref/ktruss/polbooks.csv")
+    (str(PurePath(RAPIDS_DATASET_ROOT_DIR)/"polbooks.csv"),
+     str(PurePath(RAPIDS_DATASET_ROOT_DIR)/"ref/ktruss/polbooks.csv"))
 ]
 
+
 DATASETS_SMALL = [
-    "../datasets/karate.csv",
-    "../datasets/dolphins.csv",
-    "../datasets/polbooks.csv",
+    PurePath(RAPIDS_DATASET_ROOT_DIR)/f for f in [
+        "karate.csv",
+        "dolphins.csv",
+        "polbooks.csv"]
 ]
 
+
 MATRIX_INPUT_TYPES = [
     pytest.param(
-        cp_coo_matrix, marks=pytest.mark.matrix_types, id="CuPy.coo_matrix"
+        cp_coo_matrix, marks=pytest.mark.cupy_types, id="CuPy.coo_matrix"
```

Comment:
looks like this may have been updated accidentally somehow?

All `cupy_types` occurrences should be `matrix_types`

Reply:
That was definitely a mistake when I was copying code back and forth from my machine to the docker container. I probably copied and old version